crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/tests.rs

```diff
@@ -137,7 +137,7 @@ async fn transitive_peer_gossip() {
     });
 
     // let gossip do its thing
-    tokio::time::sleep(tokio::time::Duration::from_millis(500)).await;
+    tokio::time::sleep(tokio::time::Duration::from_millis(3500)).await;
```

Comment:
This is sad, it was already needing ~50 iterations to succeed so slowing the gossip loop down now means this test becomes quite slow. It really ought to be checking for a condition instead of sleeping here anyway